target/linux/realtek/dts-5.15/rtl8380_zyxel_gs1900-8.dts

```diff
@@ -4,7 +4,7 @@
 
 / {
 	compatible = "zyxel,gs1900-8", "realtek,rtl838x-soc";
-	model = "ZyXEL GS1900-8 Switch";
+	model = "Zyxel GS1900-8v1+v2 Switch";
```

Comment:
Why did you update the vendor name capitalization? It also doesn't match the one from the Makefile anymore.

I would prefer the form "GS1900-8 v1/v2" using a forward slash. "+" does not appear to be used for this purpose elsewhere.

Reply:
Will update the variant as per you suggestion.

Regarding the vendor name, you mean here: https://github.com/openwrt/openwrt/blob/8753022aeae14eaee02e0c1e59244c5b36e454db/target/linux/realtek/image/common.mk#L60

I updated, due to the fact that is no longer ZyXEL but Zyxel due to their rebranding, but overlooked the reference in the Makefile.

I was going to ask anyway is a recasing everywhere would be something useful? Then would propose this for a separate PR and keep the casing for now.
What do you prefer?

Reply:
A translation of "ZyXEL" to "Zyxel" sounds like a lot of unnecessary churn to be honest... Old devices did use the old branding, so one could even consider such a global translation to be "wrong". You could open a separate PR for this, maybe others hold a different opinion.

You could use the new branding (in `DEVICE_ALT0_VENDOR`) for the new revision, but that results in unexpected sorting in `make menuconfig`

Reply:
Thanks for the suggestion, will update accordingly.